server/main.py

```diff
@@ -59,14 +59,17 @@ async def calculate_spectrum(payload: Payload):
             databank=payload.database,
             use_cached=True,
         )
+        if simulate_slit is not None:
+            spectrum.apply_slit(payload.simulate_slit, "nm")
+        else:
+            pass
```

Comment:
I would remove this "else pass" to keep the code more minimal so it's easier to read